src/EditorFeatures/Core.Wpf/InlineRename/InlineRenameService.cs

```diff
@@ -26,7 +27,7 @@ internal class InlineRenameService : IInlineRenameService
         private readonly IAsynchronousOperationListener _asyncListener;
         private readonly IEnumerable<IRefactorNotifyService> _refactorNotifyServices;
         private readonly ITextBufferFactoryService _textBufferFactoryService;
-
+        private readonly IFeatureServiceFactory _featureServiceFactory;
```

Comment:
This is the most generic name i've ever seen for an interface :-)  i literally have no iddea what it does.  given that it seems to eb from VS, i guess there's not much you can do aobut that.

Reply:
[https://github.com/dotnet/roslyn/issues/30542](https://github.com/dotnet/roslyn/issues/30542)

---
In reply to: [223927483](https://github.com/dotnet/roslyn/pull/29016#discussion_r223927483) [](ancestors = 223927483)

Reply:
[https://github.com/dotnet/roslyn/issues/30542](https://github.com/dotnet/roslyn/issues/30542)

---
In reply to: [222070163](https://github.com/dotnet/roslyn/pull/29016#discussion_r222070163) [](ancestors = 222070163)